src/Parsers/NvdFeedParser.php

```diff
@@ -114,7 +114,7 @@ private static function parseCveItem(\stdClass $cveItem): ?CveDetails
 
         if (isset($cveItem->impact->baseMetricV3->cvssV3->baseScore)) {
             $baseScore = $cveItem->impact->baseMetricV3->cvssV3->baseScore;
-        } else if (isset($cveItem->impact->baseMetricV2->cvssV2->baseScore)) {
+        } elseif (isset($cveItem->impact->baseMetricV2->cvssV2->baseScore)) {
```

Comment:
Huh, interesting. 👍

Reply:
only in PHP you can write an incorrect language construct and it still works :man_shrugging: